### ISSUE TITLE
elogind freezing cinnamon

### ISSUE THREAD

**Issue by cromerc** [manjaro/packages-openrc#28]
I installed elogind which also helps make plymouth work with OpenRC. However I am getting random freezes in cinnamon where the interface is completely frozen. The mouse is the only thing that moves but nothing else can be done nor refreshed. The other tty consoles work just fine and I can stop and restart cinnamon.

Maybe a fix might be in elogin 219.14? I noticed that upstream is 2 versions ahead of this repo.

**Comment by udeved** [comment]
It is perhaps related to https://github.com/wingo/elogind/issues/2
However, 219.14 also show this behaviour.

I regard the elogind package highly experimental, I just use it to test wayland session and powermanagement issues in some desktop environments.

**Comment by cromerc** [comment]
Actually it looks like I am mistaken. After uninstalling elogind and plymouth, I thought the problem was fixed. It just happened again a few moments ago. So the problem has to be with cinnamon not with elogind. I took an update to cinnamon last week around the same time I installed elogind and plymouth, so most likely something new in cinnamon is broken. I am going to roll back my packages to that day and see if the problem disappears or not.
